src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseApprovalService.cs

```diff
@@ -140,16 +141,22 @@ public class ReleaseApprovalService : IReleaseApprovalService
                     var userReleaseRoles =
                         await _userReleaseRoleService.ListUserReleaseRolesByPublication(ReleaseRole.Approver,
                             release.Publication.Id);
-
-                    var notifyReleaseApprovers = request.ApprovalStatus == ReleaseApprovalStatus.HigherLevelReview &&
-                                                 userReleaseRoles.Any();
                     
+                    
+                    var userPublicationRoles = await _context.UserPublicationRoles
+                        .Where(upr => upr.PublicationId == release.PublicationId && (upr.Role == PublicationRole.Owner || upr.Role == PublicationRole.ReleaseApprover))
+                        .ToListAsync();
+                    
+
+                    var notifyHigherReviewers = request.ApprovalStatus == ReleaseApprovalStatus.HigherLevelReview &&
+                        userReleaseRoles.Any() || userPublicationRoles.Any();
+
                     var releaseStatus = new ReleaseStatus
                     {
                         Release = release,
                         InternalReleaseNote = request.LatestInternalReleaseNote,
                         NotifySubscribers = notifySubscribers,
-                        NotifyReleaseApprovers = notifyReleaseApprovers,
+                        NotifyHigherReviewers = notifyHigherReviewers,
```

Comment:
To me, `NotifySubscribers` makes sense, i.e. whether the user decided to notify subscribers or not, regardless of whether there are any subscribers.

`NotifyHigherReviewers` is not based on the user deciding anything, but on whether any user has those roles.

These fields work differently, one audits the users decision so we can display it in the UI, one audits whether emails were sent.

I raised a discussion on this because either the naming could be better to make the reader  realise they mean different things, or we might not need `NotifyHigherReviewers` at all.

Reply:
Yep good point, as discussed after standup I've gotten rid of this field altogether